app/javascript/src/components/Dashboard/Contacts/Table/index.jsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
```

Comment:
This is a very bad practice. The purpose of adding Eslint rules and enforcing them via git hooks is to ensure that you stick onto the code standards enforced within the project. This line essentially bypasses all those rules. A firewall is created in order to prevent unwated network traffic. If we disable the firewall, then what protection do we have? Not a great analogy. But I hope you get this gist.

Bypassing git hooks or disabling Eslint rules is one of the worst things any developer can do. If there is a genuine need to bypass, then you should be taking permission from the Tech Lead of the project before doing so.

Please lemme know why exactly you have added this line. It's not necessary either ways. We can remove it.

Reply:
![Screenshot 2022-09-16 at 4 40 11 PM](https://user-images.githubusercontent.com/66865722/190626189-ff038dc9-7228-4a07-9fe0-bf55e24ca5de.png)
I was getting this error, stating `Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>``

As we don't have any code block before the return statement, VSCode wants me to replace the

```
const Table = () => {
   return ()
 }
```
 
 to 
 
```
 const Table = () => (
 )  
```

Reply:
But yes I got the gist. Definitely a bad practice to disable eslint for the entire file. Will keep this in mind henceforth.

Reply:
@adityamahajan-io https://www.loom.com/share/b06d545c18cc49c0b36ac0532b749a48

Reply:
Got it. I'm making the required changes now. I was thinking along the lines that I should use the syntax with the `return` keyword to maintain readability, but that's just my personal preference.

Reply:
@yedhink Unrelated question, but what chrome plugin are you using that shows these tags above your navbar? And what does it do?

(Screenshot from your loom video above)
<img width="782" alt="Screenshot 2022-09-16 at 7 27 01 PM" src="https://user-images.githubusercontent.com/66865722/190656165-cc383994-a35c-4e12-965e-4546b6b55c83.png">

Reply:
@adityamahajan-io It's an inbuilt feature from Chrome itself. Refer: https://blog.google/products/chrome/manage-tabs-with-google-chrome/

Can you also create a loom video and share it in the CH2022 channel? Content in the video:
- What is the Chrome group tabs?
- How to add tabs to a group? Show an example from UI.
- How to rearrange(reorder position) groups?
- How to delete a group?
- How to move items between group? Show drag and drop.

This will certainly help others.

Reply:
Sure! I will do that. Can I also mention a few more macOS menubar utilities that I use, such as "Rectangle" for window snapping, [CopyClip](https://apps.apple.com/in/app/copyclip-clipboard-history/id595191960?mt=12) for clipboard management etc

Reply:
@adityamahajan-io Don't mention `Rectangle`. Let's introduce that later. Right now I want folks to get acquainted with the basis macOS window management and understand what all are the limitations/pain points. Most would find the default window management fine. But Rectangle is for the advanced users. Right now it will be a big jump for many is CH2022.

>[CopyClip](https://apps.apple.com/in/app/copyclip-clipboard-history/id595191960?mt=12) for clipboard management etc

This is fine. But don't overwhelm with too many suggestions.

I would suggest keeping the video to just Chrome tab groups. This way, if the video is good, then I can directly post that video into our handbook. You can mention other utilities that you use under the thread as text message. Lemme know if that sounds good to you.

Reply:
That sounds good. I will record the video tonight and post it on Monday.